.github/workflows/ci.yml

```diff
@@ -1,9 +1,36 @@
-name: Native Image
+name: CI
 on:
   push:
+    branches:
+      - master
   pull_request:
+  
 jobs:
-  build:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [openjdk@1.8, openjdk@1.11]
+        scala: ["2.11", "2.12", "2.13"]
```

Comment:
Can we run cross-platform tests (including windows, ubuntu, and macOS)?

Since Github actions's concurrent limit is 20 https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits, it might be better to run cross scala version tests by `sbt +ci-test` instead?

Reply:
I agree with not having a cross build per scala version.

Reply:
It would also be nice to run tests on windows (no need for macos) but it can be done separately since it requires a lot of work sometimes.

Reply:
`sbt +ci-test` will run tests only on 2.13 because of subprojects configuration. Now we have 9 parallel jobs, so there is no need to tweak it for concurrency limit (yet).

> I agree with not having a cross build per scala version.

Why? Sometimes cross build tests is useful. With their help I found several bugs on PR stage.

Reply:
The `ci-test` command can be tweaked to run `+tests/test` instead.

https://github.com/scalameta/scalafmt/blob/2957451116d15276bad4a16c460b93ef76202079/build.sbt#L59

> Why? Sometimes cross build tests is useful. With their help I found several bugs on PR stage.

The CI should absolutely run tests for all Scala versions, my comment was rather "there is no need to have a custom CI job per Scala version". It's fine to run 2.11/2.12/2.13 tests in the same job.

Reply:
For example, in Metals we run `+cross/test` in the same CI job that runs ~600 test cases against seven different Scala versions and we haven't felt the need to split it into multiple jobs yet. One nice thing we do however is to encode the Scala version number in the test name so that it's easier to see what Scala version the test failed on.

Reply:
Done